src/snowflake/cli/api/entities/application_package_entity.py

```diff
@@ -31,3 +34,20 @@ def action_bundle(self, ctx: ActionContext):
         compiler = NativeAppCompiler(bundle_context)
         compiler.compile_artifacts()
         return bundle_map
+
+    @classmethod
+    def get_existing_app_pkg_info(
+        cls,
+        sql_executor: SqlExecutionMixin,
+        package_name: str,
+        package_role: str,
```

Comment:
We'll have to watch the number of arguments these functions eventually get, but I guess since both the entity logic and the manager logic have to invoke this, it's not worth defining a single contract, right?

Reply:
I think it turned out to be reasonable as it is